installer/dev/WindowsAppRuntimeInstall.vcxproj

```diff
@@ -96,6 +100,7 @@
       <FunctionLevelLinking>true</FunctionLevelLinking>
       <IntrinsicFunctions>true</IntrinsicFunctions>
       <PreprocessorDefinitions>NDEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <AdditionalIncludeDirectories Condition="'$(Configuration)">$(RepoRoot)\dev\Common;$(RepoRoot)\WindowsAppSDK\installer\dev\Telemetry;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
There is a typo in here in '$(Configuration. Missing trailing quote.

Reply:
Thanks for pointing that out. Neither my local VS rebuild nor the azure build pipeline caught this issue. So, I have to find other methods to validate changes to vcxproj file before checking it in.

Reply:
This is actually a design flaw on our end because our build from ProjectReunionInternal checks out this repo to build the VSIX and Installer solutions. 

You would have to change the branch ref to the feature branch on the ProjectReunionInternal repo and then run that change to our internal pipelines to verify it.

The right mitigation would be to create a new pipeline that is available publicly to build these solutions using the latest released WinAppSDK package.